### DISCUSSION TITLE
batch processing with a video instead of a directory

### DISCUSSION THREAD

**Discussion by emersound** [Mikubill/sd-webui-controlnet#2118] [Ideas]
what would be the issues with using a video file as the input directory location for batch processing?

**Comment by 2blackbar** [comment]
[EXTRACT_VID_TO_IMAGE.zip](https://github.com/Mikubill/sd-webui-controlnet/files/12623423/EXTRACT_VID_TO_IMAGE.zip)
python gui, should get you covered, its simple, you need ffmpeg.exe in path  env variables or in the same directory as this bat file

**Reply by emersound** [reply]
cool, that's a neat tool.
